unleash-yggdrasil/src/lib.rs

```diff
@@ -1760,4 +1693,52 @@ mod test {
         assert!(targeted_toggle.enabled);
         assert_eq!(targeted_toggle.variant.name, "theselectedone");
     }
+
+    #[test]
+    fn invalid_toggles_do_not_affect_other_toggles() {
+        let raw_state = r#"
+        {
+            "version": 2,
+            "features": [
+              {
+                "name": "Should_always_be_off",
+                "enabled": true,
+                "strategies": [
+                  {
+                    "name": "userWithId",
+                    "parameters": {
+                      "userIds": "[\"this\",\"is\",\"broken\"]"
+                    }
+                  }
+                ]
+              },
+              {
+                "name": "This_should_be_okay",
+                "enabled": true,
+                "strategies": [
+                  {
+                    "name": "userWithId",
+                    "parameters": {
+                      "userIds": "this,is,okay"
```

Comment:
the broken part about this is that it's an array and not a comma-separated list?

Reply:
Yessir!

We expect this format: "thing,otherthing,yetanotherthing"

The array encoded format is just completely broken